experiments/conductor/cmd/runner/runner.go

```diff
@@ -341,10 +343,11 @@ func RunRunner(ctx context.Context, opts *RunnerOptions) error {
 		}
 	case cmdPushBranch: // 9
 		processors := []BranchProcessor{
-			{Fn: makeReadyPR, CommitMsgTemplate: "make ready-pr"},
+			{Fn: runAPIChecks, CommitMsgTemplate: "{{kind}}: Normalize api checks"},
```

Comment:
APICheck is different from "make ready-pr." It is a tool that helps developers identify missed reference fields, sensitive fields, acronyms, etc. Normally, I don't think this check should be part of the PR. Instead, developers should correct the differences in the golden output rather than simply overriding it.

Reply:
cmd=26 runs and fixes API checks. This is to catch if the user missed that step so that it shows up in the PR and the reviewer can comment. It only takes a minute to run this. So more of a fail safe.

Reply:
I believe the question is: If the user forgets to run cmd 26, should the PR override the golden test output, or should it just fail the test? I prefer the latter.

Reply:
Hmm there is a case where we dont want to fix the reference and instead add it to the golden test output. Atleast for one of my PRs where the reference object is not implemented in TF or KCC and does not have create/delete/list methods. 

Like i said its a failsafe and alerts the reviewer.